### ISSUE TITLE
Horizontal subnav - aria-label should be "Local"

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-react#7664]
https://github.com/patternfly/patternfly-react/blob/2c24a2544447431362f699359ef2ef840b095db6/packages/react-core/src/components/Nav/Nav.tsx#L162

Horizontal subnav replaced tertiary nav, so seems like it should use "Local", too? That's how it is in the core examples:
* https://www.patternfly.org/v4/components/navigation/html#horizontal-subnav
* https://www.patternfly.org/v4/components/navigation/html-demos#horizontal-subnav

**Comment by nicolethoen** [comment]
Not sure - @tlabaj and @wise-king-sullyman, do either of you think this would constitute a breaking change? To change the default aria label on a component?

**Comment by wise-king-sullyman** [comment]
@nicolethoen I *think* it would, since consumers could be relying on the behavior as it is now. That being said though I'm not sure how significant to users this change would actually be.

 If nothing else though I imagine this would be likely to break some tests for consumers.

**Comment by tlabaj** [comment]
This is a tough call.  Yes it would be breaking, but the behavior now is also considered a bug. Like @wise-king-sullyman this could potentially break some consumers automation.
@mcarrano @mcoker how do you feel about holding off on this update.  If we fix it now, is it ok to break consumers testing?

**Comment by mcoker** [comment]
@tlabaj IMO either is fine, the bug is low impact and seems like the breaking change would be, too. Do we have any sort of guidelines around what's considered a breaking change regarding automation?
